lib/tasks/assignments.rake

```diff
@@ -3,6 +3,37 @@ namespace :db do
   desc 'Create assignments'
   task :assignments => :environment do
     # Assignments
+
+    rule = GracePeriodSubmissionRule.new
+    assignment_stat = AssignmentStat.new
+    puts 'Assignment 0: Grace Token usage'
+    Assignment.create(
+      short_identifier: 'A0',
+      description: 'Variables and Simple Operations',
+      message: 'using basic operators and assigning variables',
+      group_min: 2,
+      group_max: 3,
+      student_form_groups: true,
+      group_name_autogenerated: true,
+      group_name_displayed: false,
+      repository_folder: 'A0',
+      due_date: 12.hours.ago,
+      allow_web_submits: true,
+      display_grader_names_to_students: false,
+      submission_rule: rule,
+      assignment_stat: assignment_stat,
+      allow_remarks: false,
+      enable_test: false,
+      tokens_per_period: 0,
+      token_start_date: DateTime.now,
+      token_period: 1
+    )
+    p = Period.create(submission_rule: rule, hours: 3)
+    (1..3).each do |item|
```

Comment:
because there are groups now, there are now multiple periods to deduct more than one grace credit per assignment. (This is relative to the submission time)

So now there's 4 periods of 3 hours each as opposed to the original single period of 10 hours